dune/grid/CpGrid.hpp

```diff
@@ -565,23 +567,28 @@ namespace Dune
         // loadbalance is not part of the grid interface therefore we skip it.
 
         /// \brief Distributes this grid over the available nodes in a distributed machine
+        /// \param ecl Pointer to the eclipse state information. Default: null
```

Comment:
I added some more documentation about the eclipse state information
passed to loadbalance.
-- 
Join us at the DUNE User Meeting Sept. 28-29, 2015, Heidelberg, Germany
http://users.dune-project.org/projects/dune-user-meeting-2015/wiki